charts/rancher-vsphere-csi/Chart.yaml

```
@@ -21,4 +21,4 @@ maintainers:
 name: rancher-vsphere-csi
 sources:
 - https://github.com/kubernetes-sigs/vsphere-csi-driver
-version: 3.1.2-rancher2
+version: 3.2.2-rancher2
```

The 3.2.2 part is to match the upstream release version, and the `-rancherx` part represents the  revision of the chart, 
```suggestion
version: 3.1.2-rancher3
```

---

I've just rebased and changed version of Chart